packages/dids/src/methods/did-dht.ts

```diff
@@ -386,6 +402,7 @@ export enum DidDhtVerificationRelationship {
  * {@link DidDhtRegisteredKeyType | registered key type}.
  */
 const AlgorithmToKeyTypeMap = {
+  X25519    : DidDhtRegisteredKeyType.X25519,
   Ed25519   : DidDhtRegisteredKeyType.Ed25519,
   ES256K    : DidDhtRegisteredKeyType.secp256k1,
   ES256     : DidDhtRegisteredKeyType.secp256r1,
```

Comment:
@thehenrytsai minor nit but can you insert `X25519` at the bottom so that the keys remain alphabetically sorted?

Reply:
Sure, done.